cmd/kubectl-gadget/traceloop.go

```diff
@@ -172,6 +181,44 @@ func capDecode(caps uint64) (out string) {
 	return
 }
 
+func runTraceloopStart(cmd *cobra.Command, args []string) {
+	contextLogger := log.WithFields(log.Fields{
+		"command": "kubectl-gadget traceloop start",
+		"args":    args,
+	})
+
+	traces, err := utils.ListTracesByGadgetName("traceloop")
+	if err != nil {
+		contextLogger.Fatalf("Error checking traces: %v", err)
+	}
+
+	if len(traces) != 0 {
+		contextLogger.Fatalf("traceloop already running. Run 'kubectl gadget traceloop stop' before.")
```

Comment:
TODO for another PR: what if traceloop is started but then a new node is added to the cluster? It would not be possible to start traceloop there without stopping it on all nodes first.

This kind of problem should be solved with a GlobalTrace CR.